Gradle_Build_Cache_Deep_Dive/exercise1/lab/app/build.gradle.kts

```diff
@@ -31,7 +31,7 @@ application {
     mainClass.set("com.gradle.lab.App")
 }
 
-tasks.named<Test>("test") {
```

Comment:
There is a difference between the two configurations here:
- The left one says "I want to update the config for the task named test of type Test"
- The right one says "I want to update the config for all tasks of type Test"

In this particular case I want the attendees to be familiar with the former option. Its good to know the way to update configuration for all tasks of a certain type but that is not the goal here.

Reply:
I understand your point.
Thanks for explanation.